platform/administer/templates/parameters.mdx

```diff
@@ -0,0 +1,422 @@
+    size: "{{ .Values.storage_size }}"
+  {{- end }}
+```
+
+### String manipulation and defaults
+
+Use Go template functions for string manipulation and providing fallback values:
+
+```yaml
+metadata:
+  name: "{{ .Values.cluster_name | lower }}"
+  labels:
+    # Use default value if parameter is empty
+    cost-center: "{{ .Values.cost_center | default "engineering" }}"
+    # Combine multiple parameters
+    full-name: "{{ .Values.project_name }}-{{ .Values.environment_type }}"
+    # Transform string values
+    normalized-name: "{{ .Values.cluster_name | lower | replace "_" "-" }}"
+```
+
+### Working with complex parameter types
+
+#### Array parameters
+
+```yaml
+# Parameter definition
+- variable: enabled_features
+  label: Enabled Features
+  description: Select features to enable
+  type: array
+  options:
+    - ingress
+    - monitoring
+    - logging
+    - backup
+  section: Features
+
+# Template usage
+{{- range .Values.enabled_features }}
+{{- if eq . "ingress" }}
+networking:
+  ingress:
+    enabled: true
+{{- end }}
+{{- if eq . "monitoring" }}
+monitoring:
+  enabled: true
+{{- end }}
+{{- end }}
+```
+
+#### Object parameters
+
+```yaml
+# Access nested object values
+metadata:
+  labels:
+    team: "{{ .Values.team_config.name }}"
+    contact: "{{ .Values.team_config.email }}"
+```
+
+## Parameter validation and error handling
+
+### Input validation strategies
+
+#### Format validation with regular expressions
+
+```yaml
+- variable: kubernetes_version
+  label: Kubernetes Version
+  description: Kubernetes version (e.g., 1.28.4, 1.27.8)
+  type: string
+  required: true
+  validation: "^1\.(2[4-9]|[3-9][0-9])\.[0-9]+$"
+  section: Configuration
+```
+
+#### Range validation for numbers
+
+```yaml
+- variable: worker_nodes
+  label: Worker Nodes
+  description: Number of worker nodes (1-20)
+  type: number
+  required: true
+  min: 1
+  max: 20
+  section: Scaling
+```
+
+#### Complex validation with multiple constraints
+
+```yaml
+- variable: storage_class
+  label: Storage Class
+  description: Kubernetes storage class for persistent volumes
+  type: string
+  required: true
+  options:
+    - fast-ssd
+    - standard
+    - archive
+  validation: "^(fast-ssd|standard|archive)$"
+  section: Storage
+```
+
+### Error handling in templates
+
+Provide meaningful error messages and fallbacks in your templates:
+
+```yaml
+spec:
+  controlPlane:
+    # Validate required parameters
+    {{- if not .Values.environment_type }}
+    {{- fail "environment_type parameter is required" }}
+    {{- end }}
+    
+    # Provide reasonable defaults
+    replicas: {{ .Values.replica_count | default 1 }}
+    
+    resources:
+      limits:
+        # Ensure resource values are valid
+        cpu: >-
+          {{- if .Values.cpu_limit -}}
+          {{ .Values.cpu_limit }}
+          {{- else if eq .Values.environment_type "production" -}}
+          2000m
+          {{- else -}}
+          1000m
+          {{- end }}
+```
```

Comment:
Similar comment as above. If this is intended to be a generic templating example, it resembles `vcluster.yaml` closely without matching the spec and could cause confusion